### DISCUSSION TITLE
如何修改查词面板的高度呢？

### DISCUSSION THREAD

**Discussion by AIEntropy** [crimx/ext-saladict#1593] [Q&A]
![image](https://user-images.githubusercontent.com/89230575/145703887-b762caea-005e-4d41-8c6b-a1a8b8b690ab.png)
渴望得到这样比较小的，一个搜索框，这样就可以实现在视频中遇到不会的单词，可以直接使用自己已经设置好的查词面板快捷键，调出窗口完成单词查询。
想求求大佬们，出个法子叭！

**Comment by AIEntropy** [comment]
![image](https://user-images.githubusercontent.com/89230575/145703988-d78dc9ca-e87c-439b-b7d0-b8c9e5545264.png)
就是这样的

**Reply by crimx** [reply]
这个跟高度有什么关系呢？需要的时候用快捷键唤出不是更方便？不会遮挡视频。

**Reply by AIEntropy** [reply]
emmmmm我现在是已经设置了快捷键，但就是觉得这个下面的空白处，太大了，感觉很不舒服，同时有的时候要搜索的内容也可能会被挡住。

**Reply by crimx** [reply]
明白，打开[设置](https://saladict.crimx.com/manual.html#open-setting) 快捷查词，打开独立查词设置，启动记住位置与大小

**Reply by AIEntropy** [reply]
噢！谢谢啦。之前一直都没有看到！非常感谢，现在感觉看起来舒服多了。

**Reply by AIEntropy** [reply]
我想到了，这个查询面板不是可以自定义查询面板样式嘛？那岂不是我只要，把白板的样式背景设置透明就好了，这样总体看上去还是只有应该搜索框，然后又能比较好的得进行单词查询。

**Reply by AIEntropy** [reply]
> 明白，打开[设置](https://saladict.crimx.com/manual.html#open-setting) 快捷查词，打开独立查词设置，启动记住位置与大小

![image](https://user-images.githubusercontent.com/89230575/145708367-b3ac71bd-6faf-4ae0-b69b-a4f454f9c552.png)
![image](https://user-images.githubusercontent.com/89230575/145708392-bbd67a02-2498-4490-86a2-57e4bf7837d3.png)
我刚刚猛然发现，我之前用的插件好像是盗版的，是一个叫万能翻译的插件，然后我就去谷歌上商店重新下载了沙拉查词，可是当我下完以后，有一个设置窗口出现位置的选项好像修改了也不起作用，是版本有问题嘛？
